examples/train.py

```diff
@@ -202,7 +202,7 @@ def setup_exps_rllib(flow_params,
             flow_params, n_cpus, n_rollouts,
             policy_graphs, policy_mapping_fn, policies_to_train)
 
-        ray.init(num_cpus=n_cpus + 1)
+        ray.init(num_cpus=n_cpus + 1, object_store_memory=200 * 1024 * 1024)
```

Comment:
This is 0.2 GB. Is this sufficient?

Reply:
has been sufficient for local testing for me, though we can increase if necessary

Reply:
either way this is not the right PR to make this change, removing it